### PR TITLE
server: add option to specify hostname

### PR DESCRIPTION
**Pull request by yanniszark**
Signed-off-by: Yannis Zarkadas <yanniszark@arrikto.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Following environment variables are used by the software.
 * **CLIENT_SECRET** Client secret for your application.
 
 **Optional**
+* **HOSTNAME** Hostname to listen for requests. Defaults to all interfaces (0.0.0.0).
```
Comment (apyrgio) [inline README.md]: Minor, but  I'd reword it like so:

> Defaults to all IPv4/6 interfaces (0.0.0.0, ::).

so that others know that the server will bind by default on their IPv6 interfaces

**Comment by apyrgio** [comment]
Oh, one more thing. I don't suggest using `$HOSTNAME` as an environment variable, because it is an [internal Bash variable], and thus present in most installations. How about `SERVER_HOSTNAME` and `SERVER_PORT`?

[internal Bash variable]: https://www.tldp.org/LDP/abs/html/internalvariables.html

File: README.md (hunk 1 above)
Comment (yanniszark) [reply README.md]: Noted!
Will change.

**Comment by yanniszark** [comment]
@apyrgio thanks for the review!
I addressed all comments.
